trunion/validators.py

```diff
@@ -45,13 +46,17 @@ def valid_receipt(request):
     if receipt['iss'] not in request.registry.settings['trunion.permitted_issuers']:
         raise HTTPConflict("Bad issuer: \"%s\"" % receipt['iss'])
```

Comment:
As I recall, the exceptions are logged with a timestamp in production so it would be redundant.